ticdc/ticdc-debezium.md

```diff
@@ -0,0 +1,138 @@
+---
+title: TiCDC Debezium Protocol
+summary: Learn the concept of the TiCDC Debezium Protocol and how to use it.
+---
+
+# TiCDC Debezium Protocol
+
+[Debezium](https://debezium.io/) is a tool for capturing database changes. It converts each captured database change into a message called an "event" and sends these events to Kafka. Starting from v8.0.0, TiCDC supports sending TiDB changes to Kafka in Debezium format, simplifying the data link of using Debezium.
```

Comment:
I am not sure what the phrase "simplifying the data link" is intended to convey here. I would suggest changing:

> Starting from v8.0.0, TiCDC supports sending TiDB changes to Kafka in Debezium format, simplifying the data link of using Debezium.

To something like:

> Starting from v8.0.0, TiCDC supports sending TiDB changes to Kafka using a Debezium style output format, simplifying migration from MySQL databases for users who had previously been using Debezium's MySQL integration.

Reply:
That's more clear.